test/client/spec/components/typeahead.spec.jsx

```diff
@@ -26,4 +26,19 @@ describe("components/typeahead", () => {
     expect(list.containsMatchingElement(<div>bar</div>)).to.equal(true);
     expect(list.containsMatchingElement(<div>baz</div>)).to.equal(true);
   });
+
+  it("can render with a value passed down through props when in manual mode", () => {
+    const testVal = "Test Value";
+    const wrapper = shallow(<Typeahead val={testVal} manualMode />);
+    const inputVal = wrapper.find("input").props().value;
+    expect(inputVal).to.equal(testVal);
+  });
+
+  it("renders with an empty value when not in manual mode", () => {
```

Comment:
Let's add just one more test: `it("in manual mode, defaults to '' when no value is passed down", ...)`. (Should be an easy one to write, pretty similar to this one.)

That way we cover the three main cases:
* Manual mode + prop: prop renders
* Manual mode w/o prop: defaults to ""
* No-manual mode + prop: ignores prop, initializes w/ ""

(No-manual mode w/o prop (i.e., completely standard usage) doesn't seem like it needs a dedicated test here, IMO.)